samples/src/client-file-download/NamedFileDownloader.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { inject, injectable, named } from "inversify";
+
+import { ClientStorage } from "@itwin/object-storage-core";
+
+/**
+ * Simple file downloader that depends on {@link ClientStorage}. Any specific
+ * implementation of {@link ClientStorage} can be passed to the constructor
+ * making this class cloud agnostic. {@link ClientStorage} instance is resolved
+ * using instance name (which is described in the configuration as instanceName property)
+ */
+@injectable()
+export class NamedFileDownloader {
+  constructor(
+    @inject(ClientStorage)
+    @named("instanceName")
```

Comment:
Would be good to have another class here, that would use a different instance name.

Reply:
Fixed